docs/data/base/getting-started/support/support.md

```diff
@@ -52,12 +52,6 @@ You can search through existing questions and answers to see if someone has aske
 
```

Comment:
On the support page conent, I think that all open-source projects should have the same structure, unless there is a strong reason not to:

1. **Tidelift**: I don't see a reason for this to be different, so either to delete it everywhere or keep it everywhere. Since Tidelift pays us, it would make sense to me to have it.
2. **Custom work**: same incentive as 1. The soft objective was to offer people the open to shuffle the roadmap if some folks are willing to spend enough for it, but the main idea was for organizations to feel like they could spend $ to get unstuck, vs. staying stuck. I don't see strong arguments either way (remove it everywhere or add it everywhere).
3. **Social media**. Ok, I guess maybe only MUI X would make sense to post on LinkedIn? cc @joserodolfofreitas. We would then update all the other products cc @samuelsycamore 
4. **Technical support**. same incentive as 1. The problem is that MUI X paid clients ask about paid support on Base UI which isn't included in their plan today, we needed this to make it clear.

Reply:
I reverted 1, 2, 4.

On 3, I don't think there's a benefit to posting on LinkedIn about Base UI, or a benefit to linking to it. So I left that out.